floodviz/tests/test_hydrograph_utils.py

```diff
@@ -194,6 +169,12 @@ def test_valid_data_missing_count(self):
     def test_valid_data_missing_value(self):
         ret = parse_hydrodata(self.mock_data_missing_points)
         for dp in ret:
-            if dp.get('name') is 'no_data':
-                self.assertEqual(dp.get('value'), 'NA')
+            try:
```

Comment:
Yes, because of mock_data_missing_points I do know where the NA value is. I was really trying to not have to do something like ret[3] here if that's what you were thinking? It just seems like a bad idea to conform the tests to a hard coded position, when in theory that point could be at almost any position within the set.